### ISSUE TITLE
Output interpretation ?

### ISSUE THREAD

**Issue by Babak-Ebrahimi** [orian/torch-mnist-batch-normalization#1]
Dear Orian
I have ran  the code until epoch 83 and the output is like below. Could you please tell me How can I interpret such a result ? I mean that what is the train and test final error ?
(I attached the final output file)
[output.txt](https://github.com/orian/torch-mnist-batch-normalization/files/322409/output.txt)
- average row correct: 10% 
  - average rowUcol correct (VOC measure): 1.0261193662882% 
  - global correct: 10.261194029851%
    Epoch: 0084/100, cost: nan  
    <trainer> miniBatch = 72(2304 inputs) error = nan  
    ConfusionMatrix:
    [[     216       0       0       0       0       0       0       0       0       0]   100.000%  [class: 1]
    [     251       0       0       0       0       0       0       0       0       0]   0.000%    [class: 2]
    [     225       0       0       0       0       0       0       0       0       0]   0.000%    [class: 3]
    [     226       0       0       0       0       0       0       0       0       0]   0.000%    [class: 4]
    [     254       0       0       0       0       0       0       0       0       0]   0.000%    [class: 5]
    [     209       0       0       0       0       0       0       0       0       0]   0.000%    [class: 6]
    [     235       0       0       0       0       0       0       0       0       0]   0.000%    [class: 7]
    [     255       0       0       0       0       0       0       0       0       0]   0.000%    [class: 8]
    [     204       0       0       0       0       0       0       0       0       0]   0.000%    [class: 9]
    [     229       0       0       0       0       0       0       0       0       0]]  0.000%    [class: 0]

**Comment by orian** [comment]
It's train data error rates and correlation matrix.

Without GPU it's kind of slow.

In this case the net bassically got into some local minima.
